File: 900/1990A/main.java
import java.util.*;
public class main 
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while(t-- > 0)
        {
            int n = sc.nextInt();
            int arr[] = new int[51];
            
            for(int i=0;i<n;i++)
                arr[sc.nextInt()] += 1;
                
            int i=50;
            while(i >= 0 && (arr[i]&1) == 0)
                i -=1;
                
            System.out.println(i>=0?"YES":"NO");
        }
    }
}

File: 900/1990A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        int n;
        cin >> n;

        vector<int> arr(51, 0);

        for (int i = 0; i < n; i++) 
        {
            int x;
            cin >> x;

            arr[x]++;
        }

        int i = 50;
        while (i >= 0 && (arr[i] % 2 == 0))
            i--;

        cout << (i >= 0 ? "YES" : "NO") << "\n";
    }

    return 0;
}
